packaging/ios/translations.qrc

```diff
@@ -7,6 +7,7 @@
 		<file alias="subsurface_de_DE">build-ios/translations/subsurface_de_DE.qm</file>
 		<file alias="subsurface_en_GB">build-ios/translations/subsurface_en_GB.qm</file>
 		<file alias="subsurface_en_US">build-ios/translations/subsurface_en_US.qm</file>
+		<file alias="subsurface_en_US">build-ios/translations/subsurface_en_ZA.qm</file>
```

Comment:
Not a linguist, but I was under the impression that `en_ZA` is closer to `en_GB` than `en_US`? Or am I misreading this line?